src-tauri/src/main.rs

```diff
@@ -208,6 +214,20 @@ async fn main() {
         })
         .plugin(tauri_plugin_single_instance::init(|app, argv, cwd| {
             info!("{}, {argv:?}, {cwd}", app.package_info().name);
+            {
+                {
+                    app.get_window("main")
+                        .expect("Couldn't fetch window")
+                        .show()
+                        .expect("Couldn't show window");
+                    app.get_window("main")
+                        .expect("Couldn't fetch window")
+                        .set_focus()
+                        .expect("Couldn't focus window")
+                };
+                app.emit_all("single-instance", ())
+            }
+            .expect("Couldn't re-focus opened instance.");
```

Comment:
What's up with all this scoping? Can you not just have each line of `app...` call after each other?

Reply:
a brainfart.. fixed in 313926e :D